pkg/preflight/analyze.go

```diff
@@ -77,8 +77,10 @@ func doAnalyze(allCollectedData map[string][]byte, analyzers []*troubleshootv1be
 	for _, analyzer := range analyzers {
 		analyzeResult, err := analyze.Analyze(analyzer, getCollectedFileContents, getChildCollectedFileContents)
 		if err != nil {
+			strict, _ := HasStrictAnalyzer(analyzer)
```

Comment:
If you have a reason to explicitly ignore this error can you please justify it with a comment in the code. Also it may be best to log the error. Thanks

Reply:
since we have already parsed and saved the preflight spec, when we unmarshall/process the spec again, we wouldn't be having parse errors.
moreover we would want to find if the spec.analyzers[].analyze[].analyzemeta.strict is true(atleast one is true), so we can ignore the errors here.

Yes, it would be better to log.

The other way to not address erroneous conditions is to use the traverse through the Preflight struct which is very complex.